readme.txt

```diff
@@ -0,0 +1,11 @@
+===  Altapay for PrestaShop ===
+
+
+== Code Analysis==
+
+1. PHPStan is being used for running static code analysis. It's configuration file 'phpstan.neon' is available is this repository. The directories mentioned under scnDirectories option in phpstan.neon, are required for running the analysis. These directories belong to prestashop. If you don't have prestashop downloaded, you'll need to download it first and then make sure the paths are correctly reflected in phpstan.neon file. Once done, we can run the analysis: 
```

Comment:
should the folders just be empty?
From the actual task it appears that you need to download https://download.prestashop.com/download/releases/prestashop_1.7.7.0.zip and extract it to a specific folder. Please describe it well enough that someone who hasn't worked on it can do the required steps.

Reply:
added new instructions

Reply:
Much better :+1: